plugin/trino-clickhouse/src/main/java/io/trino/plugin/clickhouse/ClickHouseSessionProperties.java

```diff
@@ -28,6 +28,7 @@
         implements SessionPropertiesProvider
 {
     public static final String MAP_STRING_AS_VARCHAR = "map_string_as_varchar";
+    public static final String REPLACE_NULL_TO_DEFAULT = "replace_null_to_default";
```

Comment:
`input_format_null_as_default` is cluster level configuration in ClickHouse. Why do we need to make it configurable at session level?

Reply:
Clickhouse handling null values affects performance, as to setting its 'input_format_null_as_default' default value to True. The parameter I added is used to control the processing behavior of writing Clickhouse null values on the Trino end, and does not affect the parameters of Clickhouse itself.